src/Tribe/API/Timeline.php

```diff
@@ -0,0 +1,165 @@
+<?php
+
+namespace Tribe\HubSpot\API;
+
+use SevenShores\Hubspot\Factory;
+
+/**
+ * Class Timeline
+ *
+ * @package Tribe\HubSpot\API
+ */
+class Timeline {
+
+	protected $properties = [];
+
+	/**
+	 * Setup Hooks for Contact_Property
+	 *
+	 * @since 1.0
+	 *
+	 */
+	public function hook() {
+
+		add_action( 'tribe_hubspot_authorize_site', [ $this, 'create_event_type' ], 30 );
+
+	}
+
+	/**
+	 * Create Timeline Event Type
+	 *
+	 * @since 1.0
+	 *
+	 * @return bool
+	 */
+	public function create_event_type() {
+		//todo if the appid changes then this ids should be deleted
+
+		/** @var \Tribe\HubSpot\Admin\Settings $hubspot_options */
+		$hubspot_options = tribe( 'tickets.hubspot.admin.settings' );
+		$options         = $hubspot_options->get_all_options();
+
+		if ( ! empty( $options['eventRegistration_id'] ) ) {
```

Comment:
Curious, why are we using snake case with underscore for HubSpot admin settings?

Reply:
The spec has the name that way in HubSpot. I am going to change to underscore for our internal Id.